FAQ/FAQ-Deployment.md

```diff
@@ -83,6 +83,7 @@ echo "UUID=$UUID /mnt/usb ext4 defaults,nofail 0" | sudo tee -a /etc/fstab
 # Define `/var/lib/docker` as symbolic link to /mnt/usb
 sudo mkdir /mnt/usb/docker
 sudo ln -s /mnt/usb/docker /var/lib/docker
+mount /dev/sda1
```

Comment:
Doesn't this need to be the first command?

Reply:
A drive can only be formatted if it's not mounted, so it can't be first. 
I thought I did it in the order above but now that I look at it, I think it should be mounted after the UUID is written to the fstab. Just before creating the 'docker' folder.